src/test/java/org/apache/commons/lang3/ArrayUtilsRemoveTest.java

```diff
@@ -403,189 +403,189 @@ public void testRemoveElementShortArray() {
     @Test
     public void testRemoveAllBooleanOccurences() {
         boolean[] a = null;
-        assertNull(ArrayUtils.removeAllOccurences(a, true));
+        assertNull(ArrayUtils.removeAllOccurrences(a, true));
```

Comment:
If you create a new API (see my other comment) with the corrected name, either test both the new and old APIs next to each other, or, clone the test methods and also correct their names.

Reply:
ok，i will do it as you wish